docs/managing-minishift.md

```diff
@@ -181,8 +183,20 @@ These are not configurable at the moment and are different for each driver and t
 <a name="mounting-custom-shared-folders"></a>
 ### Mounting custom shared folders
 
-The Boot2Docker and the CentOS image include `cifs-utils`, which allows you to mount CIFS-based shared
-folders inside the VM. For example, on Windows 10 the `C:\Users` folder is shared and only needs locally
+The Minishift [Boot2Docker](https://github.com/minishift/minishift-b2d-iso) as well as the
+Minishift [CentOS](https://github.com/minishift/minishift-centos-iso) image include `cifs-utils` and
+`fuse-sshfs`. This allows you to mount host folders using [CIFS](https://en.wikipedia.org/wiki/Server_Message_Block)
+or [SSHFS](https://en.wikipedia.org/wiki/SSHFS).
+
+**Note:** When using the Boot2Docker image in conjunction with the VirtualBox driver, Virtual Box
+guest additions are automatically enabled and occupy the _/Users_ mountpoint used in the following
+examples. In this case you will need to use a different mountpoint.
+
+<a name="cifs"></a>
+#### CIFS
+
+An example for CIFS-based sharing is the mount of `C:\Users` on a Windows host into the Minishift VM.
+On Windows 10 the `C:\Users` folder is shared per default and only needs locally
```

Comment:
Not sure if should be `locally authenticated users` or `local authenticated users`